client/src/components/molecules/DonationItemForm.tsx

```diff
@@ -0,0 +1,307 @@
+                <FormItem
+                    className="item-condition-field"
+                    formItemName="Item Condition *"
+                    errorString=""
+                    isErroneous={isConditionErrorneous}
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    inputComponent={
+                        <ScrollableDropdown
+                            trigger={
+                                <TextField
+                                    input={condition ? ItemConditionToDescriptionMap.get(condition) ?? "" : ""}
+                                    isDisabled={false}
+                                    isErroneous={isConditionErrorneous}
+                                    onChange={() => {}}
+                                    name="condition"
+                                    placeholder="Select Item Condition..."
+                                    type="text"
+                                    showRedErrorText={true}
+                                    readOnly={true}
+                                    iconClassName="bi bi-caret-down-fill"
+                                />
+                            }
+                            dropdownItems={
+                                <>
+                                    <p>Select condition</p>
+                                    {Object.values(ItemCondition).map((conditionValue) => (
+                                        <span key={conditionValue} onClick={() => onConditionChange(conditionValue)}>
+                                            {ItemConditionToDescriptionMap.get(conditionValue)}
+                                        </span>
+                                    ))}
+                                </>
+                            }
+                            onDropdownClose={() => {
+                                setIsConditionDropdownOpen(false);
+                            }}
+                            onDropdownOpen={() => {
+                                setIsConditionDropdownOpen(true);
+                            }}
+                            isDropdownOpened={isConditionDropdownOpen}
+                        />
+                    }
+                />
+                <FormItem
+                    className="item-age-field"
+                    formItemName="Age of Item *"
+                    errorString=""
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    tooltipText="Please select the approximate number of years you have had this item"
+                    inputComponent={
+                        <ScrollableDropdown
+                            trigger={
+                                <TextField
+                                    input={ItemAgeToDescriptionMap.get(age) ?? ""}
+                                    isDisabled={false}
+                                    isErroneous={false}
```

Comment:
According to designs the Age of Item field should also have an error state (if no items are selected).

Reply:
Specs also said that there should be a default age selected (see [here](https://www.notion.so/uwblueprintexecs/Donation-Form-Item-Details-Step-2-a83c29169841407dace84b08de505d02#ed6888b3c8c140e89e612ab0a49dd6a1)). Since it's impossible to deselect from `ScrollableDropdown`, I didn't bother doing the error for the age field.

But now that you bring it up, I'll add an error here in case we ever decide to not use a default age